### ISSUE TITLE
Repo naming convention

### ISSUE THREAD

**Issue by ggarek** [mercari/eslint-config-mercari#5]
# TL;DR

We have a naming convention for repositories, and it is `mercari-X-Y-Z`.

# More words
Naming convention is no just fancy thing to agree on and follow.

It creates some context. As result people will spend less time guessing.

This convention, as well, provides more natural expectations on package naming if we use scopes. And it is better to use scopes.

When we install the package:

 `npm i @mercari/eslint-config`

We are able, without guessing, searching, or whatever extra effort navigate to 

`mercari-eslint-config`  
_(or `mercari/eslint-config`, plz, read to the end)_

on github.

## PS
1) The fact the `mercari` group is, so i heard, for OSS efforts, should, not affect this idea.
2) Conventions contribute a little to overall structure, whereas neglecting of such contribute to overall chaos
3) We may decide, though, that `mercari` org is already part of the name of the project _(and so it is in fact)_, then the name of the repo would be `mercari/eslint-config` and we do not need `mercari`-suffix or prefix. Maybe, that is the best option here. I would like to hear your opinion, @1000ch 

/cc @hitima

**Comment by ggarek** [comment]
One more recommendation, when you move repository, plz add a link to new location in previous repository.

**Comment by eqyiel** [comment]
I don't disagree with you about sticking to a naming convention being a good thing, but you might be interested to know that `eslint-config-hoge` is also the convention for shareable `eslint` configs:
https://www.npmjs.com/search?q=eslint-config

**Comment by ggarek** [comment]
@eqyiel yes, and it is actually for legacy reasons. Because there were no scopes before. And know we want to stick with it just because it already exists.

Better approach would be to amend eslint extension naming convention to consider 
```
@scope/eslint-config => extends: "@scope" 
```
or alike. And i think they will get to it some time later 🤔 

Well, but at least know i know the reasons lol 😅

**Comment by 1000ch** [comment]
@ggarek 

The biggest reason to changed repo name to `eslint-config-mercari` is [to reserve a namespace of npm](https://www.npmjs.com/package/eslint-config-mercari). We can get `@mercari/eslint-config` anytime if we can manage [npm @mercari org](https://www.npmjs.com/@mercari), but we cannot get `eslint-config-mercari` if reserved before we get.

Of course, making this repo open sourced is important, but the reason below is more. We can discuss how we maintain or manage.

**Comment by 1000ch** [comment]
Additional point, eslint-config's naming convention is having `eslint-config-` prefix. Then we can use `eslint-config-mercari` with a bit simpler configuration as following.

```json
{
  "extends": "mercari"
}
```

**Comment by ggarek** [comment]
OK. Understood, thank you. @1000ch 

I have comment already regarding the naming convention before:

>　...it is actually for legacy reasons. Because there were no scopes before. And know we want to stick with it just because it already exists.

But, i see the point.

Thx, for adding the link in previous repo 👍 

I think, there is no reason to keep this issue open.

**Comment by 1000ch** [comment]
Yes, closing.
